src/containers/pages/IRS/Map/tests/namibia_confgs.test.tsx

```diff
@@ -0,0 +1,129 @@
+reducerRegistry.register(IRSPlansReducerName, IRSPlansReducer);
+reducerRegistry.register(jurisdictionReducerName, jurisdictionReducer);
+reducerRegistry.register(GenericJurisdictionsReducerName, GenericJurisdictionsReducer);
+reducerRegistry.register(genericStructuresReducerName, genericStructuresReducer);
+
+/** set up data in the store needed for this view */
+const focusAreaData = superset.processData(fixtures.NamibiaFocusAreasJSON) || [];
+const jurisdictionData = superset.processData(fixtures.ZambiaAkros1JSON) || [];
+
+store.dispatch(fetchGenericJurisdictions('zm-focusAreas', focusAreaData));
+store.dispatch(fetchJurisdictions(jurisdictionData));
+
+const history = createBrowserHistory();
+
+describe('Namibia configs: components/IRS Reports/IRSReportingMap', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it('Namibia configs: renders correctly', async () => {
+    fetch.mockResponseOnce(JSON.stringify({}));
+    const mock: any = jest.fn();
+
+    const supersetServiceMock: any = jest.fn();
+    supersetServiceMock.mockImplementation(async () => []);
+
+    const focusArea = getGenericJurisdictionByJurisdictionId(
+      store.getState(),
+      'zm-focusAreas',
+      '0dc2d15b-be1d-45d3-93d8-043a3a916f30'
+    );
+
+    const structures = getGenericStructures(
+      store.getState(),
+      'zm-structures',
+      '0dc2d15b-be1d-45d3-93d8-043a3a916f30'
+    );
+
+    const jurisdiction = getJurisdictionById(
+      store.getState(),
+      '0dc2d15b-be1d-45d3-93d8-043a3a916f30'
+    );
+
+    const props = {
+      focusArea,
+      history,
+      jurisdiction,
+      location: mock,
+      match: {
+        isExact: true,
+        params: {
+          jurisdictionId: '0dc2d15b-be1d-45d3-93d8-043a3a916f30',
+          planId: (plans[0] as IRSPlan).plan_id,
+        },
+        path: `${INTERVENTION_IRS_URL}/:planId/:jurisdictionId/${MAP}`,
+        url: `${INTERVENTION_IRS_URL}/${
+          (plans[0] as IRSPlan).plan_id
+        }/0dc2d15b-be1d-45d3-93d8-043a3a916f30/${MAP}`,
+      },
+      plan: plans[0] as IRSPlan,
+      service: supersetServiceMock,
+      structures,
+    };
+    const wrapper = mount(
+      <Router history={history}>
+        <IRSReportingMap {...props} />
+      </Router>
+    );
+    await flushPromises();
+    expect(toJson(wrapper.find('.sidebar-legend-item'))).toMatchSnapshot('Legend items');
+    expect(toJson(wrapper.find('.responseItem h6'))).toMatchSnapshot('Response item titles');
+    expect(toJson(wrapper.find('.responseItem p.indicator-description'))).toMatchSnapshot(
+      'Namibia configs: Namibia configs: Response item descriptions'
+    );
+    expect(toJson(wrapper.find('p.indicator-breakdown'))).toMatchSnapshot(
+      'Namibia configs: Indicator item breakdown'
+    );
+    expect(toJson(wrapper.find('.responseItem ProgressBar'))).toMatchSnapshot(
+      'Namibia configs: Response item ProgressBar'
+    );
+    wrapper.unmount();
+  });
+});
```

Comment:
Let us add tests for:

- GisidaWrapper is called with expected props
- Superset is called with expected parameters
- It works with both points and polygons

Reply:
@moshthepitt to test points and polygons i think that would be a repetition since it is more of a functionality than UI display and it expects similar data.